packages/pulumi-aws/src/apps/storage/StorageElasticSearch.ts

```diff
@@ -0,0 +1,243 @@
+import path from "path";
+import * as pulumi from "@pulumi/pulumi";
+import * as aws from "@pulumi/aws";
+import { defineAppModule, PulumiApp } from "@webiny/pulumi-sdk";
+
+import { getAwsAccountId } from "../awsUtils";
+
+export interface ElasticSearchParams {
+    protect: boolean;
+}
+
+export const ElasticSearch = defineAppModule({
+    name: "ElasticSearch",
+    config(app, params: ElasticSearchParams) {
+        const domainName = "webiny-js";
+        const accountId = getAwsAccountId(app);
+
+        const domain = app.addResource(aws.elasticsearch.Domain, {
+            name: domainName,
+            config: {
+                elasticsearchVersion: "7.7",
+                clusterConfig: {
+                    instanceType: "t3.medium.elasticsearch",
+                    instanceCount: 2,
+                    zoneAwarenessEnabled: true,
+                    zoneAwarenessConfig: {
+                        availabilityZoneCount: 2
+                    }
+                },
+                // TODO: vpc is currently only in API
+                // vpcOptions: {
+                //     subnetIds: [vpc.subnets.private[0].id, vpc.subnets.private[1].id],
+                //     securityGroupIds: [vpc.vpc.defaultSecurityGroupId]
+                // },
+                ebsOptions: {
+                    ebsEnabled: true,
+                    volumeSize: 10,
+                    volumeType: "gp2"
+                },
```

Comment:
Another general question - how can we, within a module, access app params? For example, to check if `vpc` is enabled?

Reply:
We could return anything from the module or application `config` function. 
So if you need some additional variables, that serve informational purpose, you could return them along with app resources.